builder/oci/driver_oci.go

```diff
@@ -228,6 +229,73 @@ func (d *driverOCI) UpdateImageCapabilitySchema(ctx context.Context, imageId str
 	if err != nil {
 		return core.UpdateComputeImageCapabilitySchemaResponse{}, err
 	}
+	// no schema found, need to download the global image schema, use it as a base to create an image schema for this image
+	// and create the schema
+	if len(schema.Items) < 1 {
+		// get the global schema list
+		globalSchemaList, err := d.computeClient.ListComputeGlobalImageCapabilitySchemas(context.Background(), core.ListComputeGlobalImageCapabilitySchemasRequest{})
```

Comment:
Are there any new permissions required by the caller for this API endpoint? 

Overall these changes look good to me I just want to confirm that there are no changes needed to the required policies to support this change https://developer.hashicorp.com/packer/integrations/hashicorp/oracle#policy-reference

Reply:
Good catch!  Yes, we should update one of the policy statements since now there is the possibility that we are creating an image capability schema, not just modifying an existing one.  Can you update the following statement:

Allow group PackerGroup to use compute-image-capability-schema in tenancy

to instead read:

Allow group PackerGroup to manage compute-image-capability-schema in tenancy

Reply:
Done - thanks for confirming the needed change.